IdentityCore/src/webview/systemWebview/MSIDSafariViewController.h

```diff
@@ -24,19 +24,21 @@
 // THE SOFTWARE.
 //
 //------------------------------------------------------------------------------
+#if TARGET_OS_IPHONE && !MSID_EXCLUDE_SYSTEMWV
 
-#if !MSID_EXCLUDE_SYSTEMWV
 #import <Foundation/Foundation.h>
 #import "MSIDSystemWebviewController.h"
 
 @interface MSIDSafariViewController : NSObject<MSIDWebviewInteracting>
 
 - (instancetype)initWithURL:(NSURL *)url
+           parentController:(UIViewController *)parentController
                     context:(id<MSIDRequestContext>)context;
 
 - (BOOL)handleURLResponse:(NSURL *)url;
 
 @property (readonly) NSURL *startURL;
+@property (weak) UIViewController *parentController;
```

Comment:
nit: should this be nonatomic (it's UIKit class anyway)?

Reply:
sure